static/src/javascripts/projects/common/modules/experiments/tests/contributions-header.js

```diff
@@ -32,7 +34,8 @@ define([
         this.idealOutcome = '';
         this.canRun = function () {
             var pageObj = window.guardian.config.page;
-            return !(pageObj.isLiveBlog || pageObj.isAdvertisementFeature) && pageObj.edition === 'UK';
+            var userCookie = cookies.get('GU_U');
```

Comment:
Ok, thanks for all the feedback! For now, I'm happy with just checking for the existence of this cookie in the interest of getting it out today